fairlearn/preprocessing/_optimPreproc_helper.py

```diff
@@ -0,0 +1,393 @@
+import numpy as np
```

Comment:
Copyright note missing at the top of every file. You probably need ot run flake8 and fix all the issues.